brut.apktool/apktool-lib/src/main/java/brut/androlib/res/decoder/StringBlock.java

```diff
@@ -117,54 +272,9 @@ public String getHTML(int index) {
         if (style[1] > raw.length()) {
             return ResXmlEncoders.escapeXmlChars(raw);
         }
-        StringBuilder html = new StringBuilder(raw.length() + 32);
-        int[] opened = new int[style.length / 3];
-        boolean[] unclosed = new boolean[style.length / 3];
-        int offset = 0, depth = 0;
-        while (true) {
-            int i = -1, j;
-            for (j = 0; j != style.length; j += 3) {
-                if (style[j + 1] == -1) {
-                    continue;
-                }
-                if (i == -1 || style[i + 1] > style[j + 1]) {
-                    i = j;
-                }
-            }
-            int start = ((i != -1) ? style[i + 1] : raw.length());
-            for (j = depth - 1; j >= 0; j--) {
-                int last = opened[j];
-                int end = style[last + 2];
-                if (end >= start) {
-                    if (style[last + 1] == -1 && end != -1) {
-                        unclosed[j] = true;
-                    }
-                    break;
-                }
-                if (offset <= end) {
-                    html.append(ResXmlEncoders.escapeXmlChars(raw.substring(offset, end + 1)));
-                    offset = end + 1;
-                }
-                outputStyleTag(getString(style[last]), html, true);
```

Comment:
I don't believe `outputStyleTag` is used anymore with this removal. Can we remove this as well?

Reply:
yes, it appears so.